Frontend/ui-driver/src/Components/IndividualRideCard/View.purs

```diff
@@ -147,6 +137,30 @@ cardView push =
       ]
    ]
 
+
+specialZoneView :: forall w. (RideHistoryScreen.Action  -> Effect Unit) -> PrestoDOM (Effect Unit) w
+specialZoneView push = 
+  linearLayout
```

Comment:
If you have created a new layout then remove it's parent node
And try to reduce nodes